Formula/sqlcmd.rb

```diff
@@ -25,9 +30,9 @@ def install
   end
 
   test do
-    out = shell_output("#{bin}/sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
+    out = shell_output(bin/"sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
```

Comment:
```suggestion
    out = shell_output(bin/"sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
```
This command is not a pathname, so we should technically interpolate here.

Reply:
👍